### ISSUE TITLE
Megasync downloading strange files I can't track or stop

### ISSUE THREAD

**Issue by MaximilianKohler** [meganz/MEGAsync#390]
[**Feature request**] 
When you right click on an item, or even just mouse-over, you should be able to see the location of where the file is being downloaded to. 

I'm worried that it's overwriting files I don't want overwritten, or that it's just adding junk files somewhere I can't track. 

[**Bug**] 
The strange files:

![Misc downloads](https://user-images.githubusercontent.com/3606996/71774457-e6288f00-2f23-11ea-9363-da22663aaca1.png)

I have Megasync set to sync "G:/Documents to /Documents". And I have the download folder set to C:/User/Downloads. 

Nothing ever shows up in that downloads folder. 

I searched the file name of one of the downloads "markviewerplus" and got 0 results in both my online megasync folders and in my G drive. 

So I can't figure out what or where it's downloading.

**Comment by MaximilianKohler** [comment]
Another related problem:

I'm trying to search the cloud drive for the files that the windows program is trying to download. My intent is to manually delete them. However, when I search for a file in the cloud drive, in the results there is no way to find out where in the cloud drive the file is being stored. Clicking the "..." -> "info" only tells me Name, Size, Last modified, Date added.

**Comment by MaximilianKohler** [comment]
Oh, ok. I think I figured out the problem. Probably when megasync is not running, and you delete a file from your PC, then open Megasync and it scans your PC, instead of deleting the files you deleted it tries to download them. Probably it only deletes them if it detects them being deleted while Megasync is running?

**Comment by mattw-mega** [comment]
MEGAsync keeps your local and cloud folders in sync, so if you delete a file in one, MEGAsync will perform the corresponding deletion in the other (though, really it moves the file to the rubbish or a .debirs folder).   If MEGAsync is not running when that occurs, it will try to catch up when it is started again.    You might see an upload/download instead if you didn't have the sync enabled on restart, and then you re-enabled it, or created a new one.   If you think you are seeing a bug then please report it with the button in the settings dialog, and include logs, and let us know the exact filename of one file that didn't behave as expected, and the time at which it occurred (preferably in UTC), thanks!

**Comment by MaximilianKohler** [comment]
Ideally I would want the option to make Megasync only behave as a backup hard drive. 

If I delete files from my HD while megasync isn't running, I don't want it to redownload those files, I want it to delete them from the online backup.

**Comment by mattw-mega** [comment]
Yes that is how it works - it will remove them from your online folders when started (so long as you are not changing any sync settings).

**Comment by MaximilianKohler** [comment]
> 
> 
>  If you think you are seeing a bug then please report it with the button in the settings dialog, and include logs, and let us know the exact filename of one file that didn't behave as expected, and the time at which it occurred (preferably in UTC), thanks!

I did that, and the people who respond to support don't seem to have any expertise, and always give me generic, unrelated, and sometimes incorrect information. This goes for multiple things I've reported to support in the past. Such as the fact that Megasync prevents you from access to modifying files at some point where it's initially detecting them, and this can lead to various problems.

**Comment by rl-mega** [comment]
Please re-open the ticket if the issue persists with the latest release.
